### DISCUSSION TITLE
（β版）タグミュートの画面でユーザーのミュートのUIと合わせるかもしくは説明が欲しい気がします

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#272] [要望・アイデア・不具合]
ユーザー側のミュートの設定画面に「ミュートしたいユーザーを追加する」という入力フォームがありますが、一方でタグ側のミュート設定側には入力フォームが無いのでタグ側もユーザー側と合わせて入力フォームを設置いただくか、もしくは「タグのミュート設定は各タグのページから行うことができます」といった説明があっても良いかもしれません！（ユーザー側が「どこでミュート設定できるのだろう？」と一瞬迷うのを避けるため）

ユーザー側のミュート設定画面 :

![image](https://user-images.githubusercontent.com/25178045/183241191-5b2a6978-fea9-4391-8142-46aac7b59759.png)

タグ側のミュート設定画面 :

![image](https://user-images.githubusercontent.com/25178045/183241201-4c21c345-2302-4cdf-8b66-303e6255e865.png)

**Comment by tanaka-asami415** [comment]
@simon-ritchie 
機能を試していただきありがとうございます🙏

> タグ側もユーザー側と合わせて入力フォームを設置いただくか

こちらは、正式リリースの際に実装予定です！💪✨もうしばらくお待ちいただけますと幸いです🙇

**Reply by simon-ritchie** [reply]
@tanaka-asami415 

> こちらは、正式リリースの際に実装予定です！

承知いたしました！ご共有＆ご対応ありがとうございます・・・！🙇‍♂️

**Reply by tanaka-asami415** [reply]
@simon-ritchie 
お待たせいたしました🙇
タグミュートの正式リリースが完了し、ミュート設定画面でタグも設定できるようになりました！
💁[タグミュート機能を正式にリリースします](https://blog.qiita.com/release-tag-mute-general/)

**Reply by simon-ritchie** [reply]
おお、ご対応ありがとうございます・・・！🙇‍♂️
